flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/parameters/KubernetesJobManagerParameters.java

```diff
@@ -118,6 +118,12 @@ public Map<String, String> getRestServiceAnnotations() {
                 .orElse(Collections.emptyMap());
     }
 
+    public Map<String, String> getInternalServiceAnnotations() {
+        return flinkConfig
+                .getOptional(KubernetesConfigOptions.REST_SERVICE_ANNOTATIONS)
+                .orElse(Collections.emptyMap());
+    }
+
```

Comment:
Can you please add a test to `KubernetesJobManagerParametersTest`

Reply:
Done.